ocaml/typing/untypeast.ml

```diff
@@ -502,15 +502,16 @@ let expression sub exp =
     | Texp_try (exp, cases) ->
         Pexp_try (sub.expr sub exp, List.map (sub.case sub) cases)
     | Texp_tuple (list, _) ->
-        Pexp_tuple (List.map (sub.expr sub) list)
+        Pexp_tuple (List.map (fun e -> None, sub.expr sub e) list)
+        (* CR labeled tuples *)
```

Comment:
When leaving a CR it's good to mention what you think is wrong or will change in the future.  E.g.,

```
(* CR labeled tuples:  This should be updated to include the labels, once we have them in the typed tree. *)
```